### ISSUE TITLE
timeout exceeded { [Error: socket hang up] code: 'ECONNRESET' }

### ISSUE THREAD

**Issue by madhurangaonbit** [expressjs/body-parser#145]
HI 
I'm Getting above issue when initiating a http request from 'requestify' add on.
I used below all options to get rid of that.
But no luck so far.
```js
requestify.request(urls.ACCOUNT_AUTHENTICATION, {
        method: 'POST',
        body: reqParams,
        headers: {
          Authorization: config.AUTHORIZATION_KEY,
          'Content-Type': 'application/json',
          'Content-Length': Buffer.byteLength(reqParams)
        },
        timeout : 10000,
        dataType: 'json'
      })
```
Any help ??

PS: I heard this is the issue coming with `body-parser` That's why i placed issue here.
FYI: I'm using sails .

**Comment by dougwilson** [comment]
Hi! I just tried playing around this `requestify` and this module and has not had any issues. Your code above is leaving out way too much information for me to really help at all. Can you provide a complete server I can run and complete `requestify` code I can run? Also, please list the versions of all these modules you are using, as I am assuming you are using the most up-to-date versions, to rule out any bugs that may have been addressed.

I've never seen that particular error before, and no where in our code do we generate the message "timeout exceeded", so I'm not sure where it is coming from.

**Comment by dougwilson** [comment]
Also, I would not recommend you use `requestify`, as the last version is over 2 years old and a lot has changed with Node.js in that time. Unless you are running it with a 2 year old Node.js (0.11 or older), I recommend not even proceeding to use that module. I noticed this because it was working OK for me using Node.js 0.10.33, but when I changed to Node.js 4.2.1, `requestify` didn't work correctly any longer.
